src/macad_gym/core/maps/nav_utils.py

```diff
@@ -100,33 +103,36 @@ def get_shortest_path_waypoints(world, planner, origin, destination):
             v_current = vector(current_location, projected_location)
 
             direction = 0
-            if action == RoadOption.LEFT:
+            if action.value == RoadOption.LEFT.value:
                 direction = 1
-            elif action == RoadOption.RIGHT:
+            elif action.value == RoadOption.RIGHT.value:
                 direction = -1
-            elif action == RoadOption.STRAIGHT:
+            elif action.value == RoadOption.STRAIGHT.value:
                 direction = 0
             select_criteria = float('inf')
 
             #   Choose correct path
-            for wp_select in wp_choice:
-                v_select = vector(current_location,
-                                  wp_select.transform.location)
+            future_wp = []
+            for wp in wp_choice:
+                future_wp.append(wp.next(5)[0])
```

Comment:
Could you explain why this necessary?
Also, hard-coding `5` as in `wp.next(5)` isn't good.